src/vscode/Utils.ts

```diff
@@ -302,3 +302,12 @@ async function updateCompilerDiagnostics(diagnostics: VscodeDiags, notes: Compil
         diagnostics.set(vscode.Uri.file(file), arr)
     }
 }
+
+export function getNonce() {
+    let text = '';
+    const possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```

Comment:
I don't think it matters because it's not visible, but this will generate profanity more often than you'd think.